packages/common/src/api/tan-query/useTrackByParams.ts

```diff
@@ -18,15 +18,13 @@ type TrackParams = { trackId: ID } | { handle: string; slug: string }
  * @returns The track data or null if not found
  */
 export const useTrackByParams = (params: TrackParams, options?: Config) => {
-  const permalink =
-    'handle' in params ? `/${params.handle}/${params.slug}` : null
-  const trackId = 'trackId' in params ? params.trackId : null
+  const permalink = params.handle ? `/${params.handle}/${params.slug}` : null
```

Comment:
nit: at this point maybe just destructure params

Reply:
yeah def good call